### PR TITLE
update wdl tutorial

### PR DESCRIPTION
**Pull request by NatalieEO**

### CODE REVIEW

File: docs/getting-started/getting-started-with-services.rst [hunk 1]
```diff
@@ -22,7 +21,7 @@ Create Your Service
 The first step is to create a file named ``.dockstore.yml``. The ``.dockstore.yml``  is a configuration file used to describe your service. Technically, having a valid YAML and the right version is
 all that is required for your service to appear on Dockstore. But, we will briefly cover an example ``.dockstore.yml`` written for the Xena Hub to show what will generally be needed to create a working service.
 You can also view a template ``.dockstore.yml`` file `here
-<https://github.com/dockstore/dockstore-documentation/tree/develop/docs/assets/templates/.dockstore.yml>`_.
+<https://github.com/dockstore/dockstore-documentation/blob/1e487775aa8ca22e38ec2193dc671d31b3d1eecf/docs/assets/templates/.dockstore.yml>`_.
```
Comment (agduncan94) [inline docs/getting-started/getting-started-with-services.rst]: :+1:

File: docs/getting-started/getting-started-with-services.rst [hunk 2]
```diff
@@ -86,7 +85,3 @@ Once you have a version, you can now publish your service! Publishing will creat
 .. figure:: /assets/images/docs/services-public-page.png
 
 Now other users can view and :doc:`star <../end-user-topics/starring>` your service. You will also have the option to add published services to a :doc:`collection <../advanced-topics/organizations-and-collections>`.
-
-
-.. discourse::
-    :topic_identifier: 1970
```
Comment (agduncan94) [inline docs/getting-started/getting-started-with-services.rst]: Did you mean to remove the discourse embed?

Comment (NatalieEO) [reply docs/getting-started/getting-started-with-services.rst]: totally did not mean to. Added it back.

File: docs/getting-started/getting-started-with-wdl.rst [hunk 3]
```diff
@@ -225,7 +236,7 @@ the JSON config file:
 
 ::
 
-    $> dockstore tool launch --local-entry Dockstore.wdl --json test.wdl.json 
+    $> dockstore tool launch --local-entry Dockstore.wdl --json test.wdl.json
```
Comment (Ldcabansay) [inline docs/getting-started/getting-started-with-wdl.rst]: Isn't this a workflow launch?

File: docs/getting-started/getting-started-with-wdl.rst [hunk 4]
```diff
@@ -61,8 +65,10 @@ repository:
 ::
 
     task bamstats {
-        File bam_input
-        Int mem_gb
+        input {
```
Comment (Ldcabansay) [inline docs/getting-started/getting-started-with-wdl.rst]: This docs refers to and link out to the existing bamstats wdl, but that link isn't WDL 1.0. Ideally both would be identical, but I'm not sure if a 1.0 version exists (either registered in dockstore or as a branch). I'm kind  of grappling with this on the CWL docs too.

File: docs/getting-started/getting-started-with-wdl.rst (hunk 3 above)
Comment (coverbeck) [reply docs/getting-started/getting-started-with-wdl.rst]: They both exist.

**Pull request merged**